src/cogent3/app/evo.py

```diff
@@ -104,29 +106,142 @@ def __init__(
             using a list of dicts corresponding to edge_sets, e.g.
             ``[dict(edges=['Human', 'Chimp'], is_independent=False, upper=10)]``.
             This value is passed to <likelihood function>.set_time_heterogeneity()
-        param_rules : list
+        param_rules
             other parameter rules, passed to
             <likelihood function>.set_param_rule()
-        opt_args : dict
+        opt_args
             arguments for the numerical optimiser, e.g.
             dict(max_restarts=5, tolerance=1e-6, max_evaluations=1000,
             limit_action='ignore')
         lower, upper
             bounds for all rate and length parameters. Ignored if a
             rule in ``param_rules`` or ``time_het`` has a value defined.
-        split_codons : bool
+        split_codons
             if True, incoming alignments are split into the 3 frames and each
             frame is fit separately
-        show_progress : bool
+        show_progress
             show progress bars during numerical optimisation
-        verbose : bool
+        verbose
             prints intermediate states to screen during fitting
 
         Returns
         -------
         Calling an instance with an alignment returns a model_result instance
         with the optimised likelihood function. In the case of split_codons,
         the result object has a separate entry for each codon position.
+
+        Examples
```

Comment:
For the examples only show the actual execution and the result of execution twice. The first time when it does successfully return a result object and the second time when it returns a not completed object. In other cases just show how to configure the model app.